README.md

```diff
@@ -2206,8 +2206,11 @@ expect(Person("Susanne", "Whitley", 43, listOf()))
 ```text
 expected that subject: Person(firstName=Susanne, lastName=Whitley, age=43, children=[])        (readme.examples.Person <1234789>)
 ◆ ▶ children: []        (kotlin.collections.EmptyList <1234789>)
-    ◾ ▶ has at least one element: false
-        ◾ is: true
+    ◾ has: a next element
+      » all entries: 
+          » ▶ age: 
```

Comment:
@wordhou this looks odd, looks like we create an empty feature assertion. Could you check please?

Reply:
The problem is unrelated to your PR. Going to merge it and fix the Problem myself.

Reply:
I was just about to comment on this! It looks like the issue is somewhere in the feature extractor code, although I haven't had the time to figure out where.

Reply:
yes exactly, I spotted the problem and I am going to fix it soonish.